Sage/carbon
# chore(cypress): convert remaining support utils to typescript (FE-5895)

### Proposed behaviour

- Convert all remaining cypress support utils to TypeScript
- Correct any util typing that is causing type errors
- Rename `useJQueryCssValueAndAssert` util to `assertCssValueIsApproximately` to reflect its behaviour
- Remove unused `dragAndDrop` helper

### Current behaviour

- Still have some cypress utils written in JavaScript with no TypeScript type declarations - which will be needed when migrating the cypress `*.cy.*` tests to TypeScript
- Some utils need their types modifying - to address some type errors encountered while migrating the test files to TypeScript

### Checklist




#### QA


### Testing instructions

- Ensure `npm run type-check` passes without errors
- Ensure all cypress tests still pass
